docs/rest_api.md

```diff
@@ -173,6 +169,92 @@ Additionally, the `show_networks` api call may return the channel identifier
 Where documentation specifies that the network driver should choose a
 default channel, the VLAN drivers choose `vlan/native`.
 
+### list_networks
+
+`GET /networks`
+
+List all networks.
+
+Returns a JSON dictionary of dictionaries, indexed by the name of the network with interior dictionaries indexed by project and channel.
+
+The response must contain the following fields:
+
+* "network", the name of a network
+* "driver_id", the id of the network
+* "projects", a list of projects with access to the network or 'None' if network is public 
+
+Example Response:  
+	{
+		"netA": {
+			"driver_id": "101", 
+			"projects": ["qproj-01", qproj-02"]
+			},
+		"netB": {
+			"driver_id": "102", 
+			"projects": None}
+	}
+
+Authorization requirements:
+
+Only admins can list the networks.
+
+### list_network_attachments
+
+`GET /network/<network>/attachments`
+
+List all nodes that are attached to network <network>.
+    
+If optional argument 'project' is supplied will only list attached nodes belonging to the specified project.
```

Comment:
Where does this argument come from? There's no body in a GET request, is it a query parameter? I don't think our `rest_call` decorator can even do that (unless you've changed it). This needs clearer specification.

Also, please hard-wrap to 72-80 chars.